lr-cli/cortx_setup/commands/storageset/config/durability.py

```diff
@@ -64,52 +73,71 @@ class DurabilityConfig(Command):
 
     def run(self, **kwargs):
         try:
-            index = 'storage_durability_index'
+            confstore_file_index = 'storage_durability_index'
             storage_set_name = kwargs['storage_set_name']
             durability_type = kwargs['durability_type']
             cluster_id = get_cluster_id()
+            durability_dict = {
```

Comment:
Correct alignment for values inside the dictionary

Reply:
Fixed.